versions/1/_dev/deploy/tf/spec.yml

```diff
@@ -5,9 +5,22 @@ spec:
       pattern: '^*.tf$'
       type: file
       required: true
+    - description: Terraform dependency lock file
+      pattern: ".terraform.lock.hcl"
+      type: file
+      required: true
```

Comment:
Btw, this may be breaking for current packages. At least we will need to add this file to the AWS package when updating the spec there.
https://github.com/elastic/integrations/tree/8e600daead4b89e97271e4976b984722c8db7c74/packages/aws/data_stream/ec2_metrics/_dev/deploy/tf

Reply:
Nice find, actually I don't want to introduce a breaking change here. Do we have a process here? I will make this not required, but I think it make sense to have it everywhere. Is there a way to notify users of this change? For example outputting a message: "this field is going to default to true from version x.y.z"

Reply:
Maybe we can add this as optional by now, add after some time make it required. At the moment I think that only the AWS package uses it, so not many users affected :)

Reply:
Yeah, let's mark it optional for now.